CHANGELOG.md

```diff
@@ -1,3 +1,45 @@
+# 93.5.0
+
+This minor release introduces new customization APIs for Slider. The track
+height and the visibility of track tick marks can be customized.
+
+## New features
+
+`MDCSlider` allow customization of the track height.
+
+```objc
```

Comment:
I believe we traditionally put both obj-c and swift examples.

Reply:
Spoke offline we are going to just do obj-c to help this release land faster.

Reply:
Done. Looking at the CHANGELOG, it seems we generally don't do.